clustering/clustering_example.Rmd

```diff
@@ -96,7 +96,7 @@ and filename within the function to the respective file suffix.
 
 ```{r Save heatmap as a png}
 # Open a png file
-png("HeatmapGSE12955.png")
+png(file.path("results", "HeatmapGSE12955.png"))
```

Comment:
This should be in a `plots` directory that should be created in this notebook.

Reply:
The original file is also still in the top directory of this module as far as I can tell.